### PR TITLE
Min-Max-Sum Program in Python

### PR DESCRIPTION
**Pull request by Manasi2001**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Added the program for 'Min-Max-Sum' and updated the README as well.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

This PR is with reference to issue #6974.

### CODE REVIEW

**Comment by Manasi2001** [comment]
@HarshCasper @bhav09 @ankitaggarwal23 please review this PR. Thanks in advance!

File: Python/math/min_max_sum.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+MMSum(array)                           # calling the Min-Max-Sum function 
+  
+'''
+Sample Input:
+1 2 3 4 5
+
+Sample Output:
+10 14
+
+'''
```
Comment (bhav09) [inline Python/math/min_max_sum.py]: Add a new line at the end of the file.

File: Python/math/min_max_sum.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+Example:
+arr = [1,3,5,7,9]
+The minimum sum is 1+3+5+7 = 16 and the maximum sum is 3+5+7+9 = 24. So, the output will be:
+16 24
+
+'''
+
+def MMSum(a):
+    a.sort()     # sorting the array so that it becomes easier to find the min and max set of values
```
Comment (bhav09) [inline Python/math/min_max_sum.py]: Comments should be above the lines of code and not alongside.

File: Python/math/min_max_sum.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    print(minn,maxx)
+    
+user_input = (input().strip().split()) # getting the input
+array = []
+for i in user_input:
+    array.append(int(i))
+MMSum(array)                           # calling the Min-Max-Sum function 
+  
+'''
+Sample Input:
```
Comment (bhav09) [inline Python/math/min_max_sum.py:39]: Need to mention the complexity of the code that is being used.

**Review by bhav09 — CHANGES_REQUESTED** [review]
Reviewed.

**Comment by Manasi2001** [comment]
@bhav09 please give it a look again.

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**